include/filters/transfer_function.hpp

```diff
@@ -349,4 +349,4 @@ bool MultiChannelTransferFunctionFilter<T>::update(const std::vector<T>  & data_
 
 }
 
-#endif //#ifndef FILTERS_TRAjNSFER_FUNCTION_H_
+#endif //#ifndef FILTERS_TRAjNSFER_FUNCTION_HPP_
```

Comment:
It's not really in this PR, but there's an extra j in here.

```suggestion
#endif //#ifndef FILTERS_TRANSFER_FUNCTION_HPP_
```

Reply:
Fixed in 9a42564